app/jobs/create_impression_job.rb

```diff
@@ -27,7 +31,7 @@ def perform(id, campaign_id, property_id, ad_template, ad_theme, ip_address, use
       property: property,
       ad_template: ad_template,
       ad_theme: ad_theme,
-      ip_address: ip_address,
+      ip_address: Digest::MD5.hexdigest("#{ip_address}#{ip_address_salt}"),
```

Comment:
Will this break our call for Maxmind data?

Reply:
No this happens after all the Maxmind work has been performed.